plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui/src/com/google/cloud/tools/eclipse/appengine/libraries/ui/CloudLibrariesPage.java

```diff
@@ -178,12 +178,16 @@ public void setSelection(IClasspathEntry containerEntry) {
       // FIXME: read in current dependencies for maven projects
       return;
     }
-    try {
-      List<Library> savedLibraries = BuildPath.loadLibraryList(project, new NullProgressMonitor());
-      setSelectedLibraries(savedLibraries);
-    } catch (CoreException ex) {
-      logger.log(Level.WARNING,
-          "Error loading selected library IDs for " + project.getElementName(), ex); //$NON-NLS-1$
+    if (containerEntry != null) {
+      // ignore persisted library lists if this is a new container
```

Comment:
Cool!

I think the location of the comment is confusing though.

Reply:
I have to back this out: https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/2553#issuecomment-344054071